pyo3-polars/src/lib.rs

```diff
@@ -234,6 +512,100 @@ impl IntoPy<PyObject> for PyDataFrame {
     }
 }
 
+impl IntoPy<PyObject> for PyAnyValue<'_> {
+    fn into_py(self, py: Python) -> PyObject {
+        let utils = UTILS.as_ref(py);
+        match self.0 {
+            AnyValue::UInt8(v) => v.into_py(py),
+            AnyValue::UInt16(v) => v.into_py(py),
+            AnyValue::UInt32(v) => v.into_py(py),
+            AnyValue::UInt64(v) => v.into_py(py),
+            AnyValue::Int8(v) => v.into_py(py),
+            AnyValue::Int16(v) => v.into_py(py),
+            AnyValue::Int32(v) => v.into_py(py),
+            AnyValue::Int64(v) => v.into_py(py),
+            AnyValue::Float32(v) => v.into_py(py),
+            AnyValue::Float64(v) => v.into_py(py),
+            AnyValue::Null => py.None(),
+            AnyValue::Boolean(v) => v.into_py(py),
+            AnyValue::String(v) => v.into_py(py),
+            AnyValue::StringOwned(v) => v.into_py(py),
+            AnyValue::Categorical(idx, rev, arr) | AnyValue::Enum(idx, rev, arr) => {
+                let s = if arr.is_null() {
+                    rev.get(idx)
+                } else {
+                    unsafe { arr.deref_unchecked().value(idx as usize) }
+                };
+                s.into_py(py)
+            }
+            AnyValue::Date(v) => {
+                let convert = utils.getattr(intern!(py, "_to_python_date")).unwrap();
+                convert.call1((v,)).unwrap().into_py(py)
+            }
+            AnyValue::Datetime(v, time_unit, time_zone) => {
+                let convert = utils.getattr(intern!(py, "_to_python_datetime")).unwrap();
+                let time_unit = time_unit.to_ascii();
+                convert
+                    .call1((v, time_unit, time_zone.as_ref().map(|s| s.as_str())))
+                    .unwrap()
+                    .into_py(py)
+            }
+            AnyValue::Duration(v, time_unit) => {
+                let convert = utils.getattr(intern!(py, "_to_python_timedelta")).unwrap();
+                let time_unit = time_unit.to_ascii();
+                convert.call1((v, time_unit)).unwrap().into_py(py)
+            }
+            AnyValue::Time(v) => {
+                let convert = utils.getattr(intern!(py, "_to_python_time")).unwrap();
+                convert.call1((v,)).unwrap().into_py(py)
+            }
+            // TODO: need help here
+            AnyValue::Array(_v, _) | AnyValue::List(_v) => {
+                todo!();
+                // PySeries(v).to_list()
+            }
+            ref av @ AnyValue::Struct(_, _, flds) => struct_dict(py, av._iter_struct_av(), flds),
+            AnyValue::StructOwned(payload) => struct_dict(py, payload.0.into_iter(), &payload.1),
+            // TODO: Also need help here
```

Comment:
Help needed :)

Same as with the `DataType` struct, implementing the `ObjectValue` struct would lead to a lot of new code that would need to be copied.

Maybe we could just not support this feature? Open to ideas :)